packages/wonder-blocks-data/src/util/__tests__/response-cache.test.js

```diff
@@ -451,137 +451,205 @@ describe("../response-cache.js", () => {
-
-                // Act
-                cache.getEntry(fakeHandler, "options");
-
-                // Assert
-                expect(customCache.retrieve).toHaveBeenCalledWith(
-                    fakeHandler,
-                    "options",
-                );
-                expect(internalCache.retrieve).not.toHaveBeenCalled();
-            });
-
-            it("CSR should store framework entry in custom cache when custom cache misses", () => {
-                // Arrange
-                jest.spyOn(Server, "isServerSide").mockReturnValue(false);
-                const internalCache = new MemoryCache();
-                jest.spyOn(internalCache, "retrieve").mockReturnValue({
-                    data: "data!",
+            describe("SSR", () => {
+                it("should use framework cache, not custom cache", () => {
+                    // Arrange
+                    jest.spyOn(Server, "isServerSide").mockReturnValue(true);
+                    const internalCache = new MemoryCache();
+                    jest.spyOn(internalCache, "retrieve");
+                    const cache = new ResponseCache(internalCache);
+                    const customCache = {
+                        store: jest.fn(),
+                        retrieve: jest.fn(),
+                        remove: jest.fn(),
+                        removeAll: jest.fn(),
+                    };
+                    const fakeHandler: IRequestHandler<string, string> = {
+                        getKey: () => "MY_KEY",
+                        type: "MY_HANDLER",
+                        shouldRefreshCache: () => false,
+                        fulfillRequest: jest.fn(),
+                        cache: customCache,
+                        hydrate: true,
+                    };
+
+                    // Act
+                    cache.getEntry(fakeHandler, "options");
+
+                    // Assert
+                    expect(internalCache.retrieve).toHaveBeenCalledWith(
+                        fakeHandler,
+                        "options",
+                    );
+                    expect(customCache.retrieve).not.toHaveBeenCalled();
                 });
-                const cache = new ResponseCache(internalCache);
-                const customCache = {
-                    store: jest.fn(),
-                    retrieve: jest.fn(),
-                    remove: jest.fn(),
-                    removeAll: jest.fn(),
-                };
-                const fakeHandler: IRequestHandler<string, string> = {
-                    getKey: () => "MY_KEY",
-                    type: "MY_HANDLER",
-                    shouldRefreshCache: () => false,
-                    fulfillRequest: jest.fn(),
-                    cache: customCache,
-                    hydrate: true,
-                };
 
-                // Act
-                const result = cache.getEntry(fakeHandler, "options");
+                it("should not store framework entry in custom cache", () => {
```

Comment:
This and the following test case are new, to verify that the SSR behavior is correct. Without my change, these would fail.